orbitize/system.py

```diff
@@ -121,28 +127,32 @@ def __init__(self, num_secondary_bodies, data_table, system_mass,
 
             # Add epoch of periastron prior.
             self.sys_priors.append(priors.UniformPrior(0., 1.))
-            self.labels.append('epp{}'.format(body+1))
+            self.labels.append('tau{}'.format(body+1))
 
         #
         # Set priors on total mass and parallax
         #
         self.labels.append('plx')
-        self.labels.append('mtot')
         if plx_err > 0:
             self.sys_priors.append(priors.GaussianPrior(plx, plx_err))
         else:
             self.sys_priors.append(plx)
         
         if self.fit_secondary_mass:
             for body in np.arange(num_secondary_bodies):
-                    self.sys_priors.append(priors.JeffreysPrior(1e-6, 1)) # in Solar masses for onw
+                self.sys_priors.append(priors.JeffreysPrior(1e-6, 1)) # in Solar masses for now
```

Comment:
Didn't we rename this to LogUniform? Or would this be fixed when other changes are merged from master/other branches?

Reply:
(maybe this is the conflict that @semaphoreP mentions!)